pkg/linode-bs/luks.go

```diff
@@ -0,0 +1,405 @@
+/*
+Copyright 2019 linkyard ag
+Copyright cloudscale.ch
+Copyright 2022 Akamai Technologies
```

Comment:
since unfortunately they didnt export any of this stuff and you had to lift the file can you least link to the source https://github.com/cloudscale-ch/csi-cloudscale/blob/master/driver/luks_util.go and give them credit in a comment?